### DISCUSSION TITLE
AdGuard Home's DNS-over-TLS port 853 is not working

### DISCUSSION THREAD

**Discussion by SuAdnim** [AdguardTeam/AdGuardHome#4962] [Q&A]
Sorry to interrupt, but I have a question that has been bothering me for a long time.
I am using RHEL-8.6.0 system in aws ec2, I have configured ssl certificate, ec2 security group has squared port 853.
Port 853 is not working, I check that port 853 is listening, no firewall is enabled, but port 853 is always in filtered state.
I use nmap to scan 127.0.0.1 and find that port 853 is in open state, but when I use nmap to scan public ip, I find that it is in filtered state.
Is this a problem caused by incompatibility with RHEL-8.6.0? Have you encountered this problem?

**Comment by iAmBipinPaul** [comment]
I'm rerunning behind reverser proxy traefik and I get this error 

```
traefik-1    | time="2024-09-14T15:25:03Z" level=error msg="Error while connecting to backend: dial tcp 172.19.0.3:853: connect: connection refused"
```
